mvpa2/clfs/gda.py

```diff
@@ -259,3 +259,94 @@ def _g_k(self, data):
             dm = data - m
             res.append(b - 0.5 * np.sum(np.dot(dm, covi) * dm, axis=1))
         return np.array(res).T
+
+class ShrinkageLDA(GDA):
+    """LDA with Ledoit-Wolf estimate of covariance matrix
+    """
+
+    __tags__ = GDA.__tags__ + ['linear', 'lda', 'shrinkage']
+
+    def __init__(self, shrinkage_estimator='LedoitWolf', block_size=1000, \
+    **kwargs):
+        super(ShrinkageLDA, self).__init__(**kwargs)
+        self.block_size = block_size
+        if shrinkage_estimator is 'LedoitWolf':
+            self.shrinkage_estimator = LedoitWolf(block_size=block_size)
+        elif shrinkage_estimator is 'OAS':
+            self.shrinkage_estimator = OAS()
```

Comment:
better delay this instantiation until _train, and do not even assign that instance to any ShrinkageLDA attribute then.
It would allow to change that parameter of a classifier instance and retrain and would keep its `repr` correct